src/manipulators/getHandlePixelPosition.js

```diff
@@ -0,0 +1,23 @@
+import external from '../externalModules.js';
+import { state } from './../store/index.js';
+
+/**
+ * Return the handle position considering offset for touch interaction.
+ * @public
+ * @function getHandlePixelPosition
+ * @memberof Manipulators
+ *
+ * @param {Object} eventData - Data object associated with the event
+ * @param {string} interactionType - Type of user's input
+ * @returns {Object} - The translated point object
+ */
+export default function(eventData, interactionType) {
+  const { currentPoints, element } = eventData;
+  const page = currentPoints.page;
```

Comment:
can we use es6 destructuring instead (const {page} = currentPoints;) ? and the applies to handleTouchOffset

Reply:
Done!